src/SolutionTemplate/UnoSolutionTemplate.WinUI.netcore/Mobile/UnoQuickStart.Mobile.csproj

```diff
@@ -7,7 +7,7 @@
 
 		<!-- Disabled because of https://github.com/xamarin/xamarin-macios/issues/16401-->
 		<!--
-		$if$($UseAppKit$ == True)<TargetFrameworks>$(TargetFrameworks);$basetargetframework$-macos</TargetFrameworks>$endif$
+		$_if$($UseAppKit$ == True)<TargetFrameworks>$(TargetFrameworks);$basetargetframework$-macos</TargetFrameworks>$endif_$
 		-->
```

Comment:
Does the following work instead?

```xml
		$if$($UseAppKit$ == True)
		<!-- Disabled because of https://github.com/xamarin/xamarin-macios/issues/16401-->
		<!--
		<TargetFrameworks>$(TargetFrameworks);$basetargetframework$-macos</TargetFrameworks>
		-->
		$endif$
```

Reply:
I try

Reply:
I tried xml is not valid

```xml
		<!--#if (macos)

		<!-- Disabled because of https://github.com/xamarin/xamarin-macios/issues/16401--><!--
		<TargetFrameworks>$(TargetFrameworks);net6.0-macos</TargetFrameworks>
		-->

#endif -->
```

Reply:
On my side, the produced `csproj` seems to be actually valid already:

![image](https://user-images.githubusercontent.com/31348972/201127199-ede1ecf5-7632-45a0-b5b9-cb49983bf966.png)

![image](https://user-images.githubusercontent.com/31348972/201127482-82218e6d-7304-4dcc-9bad-083c507c3458.png)

Can you please show the csproj you're getting and what makes it invalid?